models/DataObject/AbstractObject.php

```diff
@@ -1443,4 +1449,24 @@ protected function getListingCacheKey(array $args = [])
 
         return $cacheKey;
     }
+
+    /**
+     * @param bool | null $o_reverseSort
+     *
+     * @return AbstractObject
+     */
+    public function setReverseSort(?bool $o_reverseSort): Element\ElementInterface
+    {
+        $this->o_reverseSort = $o_reverseSort;
+
+        return $this;
+    }
+
+    /**
+     * @return bool
+     */
+    public function getReverseSort(): bool
```

Comment:
we should also consider sort order in getChildren method here https://github.com/pimcore/pimcore/blob/c63c1b5e448503f6969bcbacd8f5ea83e9c01222/models/DataObject/AbstractObject.php#L412

Reply:
also add here for nodes info https://github.com/pimcore/pimcore/blob/ae677a1de09d1c488cce9cea3ba95a3e44954ae0/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php#L230

Reply:
do you mean like: $list->setOrder($this->getChildrenSortOrder()); instead of $list->setOrder('asc');

Reply:
yup, looks good 👍 also can you please add `'sortOrder' => $child->getChildrenSortOrder(),` here after SortBy https://github.com/pimcore/pimcore/blob/ae677a1de09d1c488cce9cea3ba95a3e44954ae0/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php#L230

Reply:
I added the sortorder in the previous commit :)